src/App.tsx

```diff
@@ -1,61 +1,118 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo, TodoUser, User } from './interfaces';
+
+const todosUsersFiltered: TodoUser[] = todosFromServer.map((todo: Todo) => {
+  const matchUser = usersFromServer.find(
+    (user: User) => user.id === todo.userId,
+  );
+
+  return {
+    ...todo,
+    user: matchUser,
+  };
+});
 
 export const App = () => {
+  const [count, setCount] = useState(16);
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [errTitle, setErrTitle] = useState(false);
+  const [errUser, setErrUser] = useState(false);
```

Comment:
Consider improving naming of this state to improve readability
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig